software/script/chameleon_cmd.py

```diff
@@ -19,13 +19,16 @@
 DATA_CMD_ENTER_BOOTLOADER = 1010
 DATA_CMD_GET_DEVICE_CHIP_ID = 1011
 DATA_CMD_GET_DEVICE_ADDRESS = 1012
-DATA_CMD_GET_GIT_VERSION = 1017
 
 DATA_CMD_SAVE_SETTINGS = 1013
 DATA_CMD_RESET_SETTINGS = 1014
 DATA_CMD_SET_ANIMATION_MODE = 1015
 DATA_CMD_GET_ANIMATION_MODE = 1016
 
+DATA_CMD_GET_GIT_VERSION = 1017
+
```

Comment:
1018 is missing

Reply:
1018 used to get current selected slot

Reply:
> 1018 used to get current selected slot

Yeah, but it should probably be also implemented in the cli, just for the sake of completeness.

Reply:
we have a lot of unimplemented methods in cli, method to get currently selected slot is useless if we can't even change it